### PR TITLE
[CTD-264] create Vyos build

### PR DESCRIPTION
**Pull request by Digital-Larry**
Vyos build required following changes:
- Adjusted versions of package dependencies to match Vyos (Debian 6 level)
- Added dependency on cron (which was CTD-205 but applies to Vyos as well)
- updated build.sh to make 32 and 64 bit Debian builds using "etch" binaries
- updated build.sh to allow a "tag" so we can have Debian packages with the same Debian architecture but different daemons inside to accommodate older OS

### CODE REVIEW

**Comment by Digital-Larry** [comment]
I have tested the 64-bit package on a Vyos AWS EC2 instance.
Checked interactive installer, bulk registration, and bulk scripting.

File: scripts/build.sh [hunk 1]
```diff
@@ -158,22 +165,25 @@ build() {
...
         mv "$pkgFolder".deb.tar "$cwd/$filename"
 
     fi
+    ls -l "$cwd/$filename"
 
 }
 
 buildDeb=0
 arch="armhf"
+tag=""
```
Comment (danawoodman) [inline scripts/build.sh]: Is there a way for us to not repeat all these empty `tag=""`? Could we instead default to an empty string if the arg isn't present?

File: connectd/usr/bin/connectd_startup_control [hunk 2]
```diff
@@ -6,13 +6,13 @@
 #  installed remote.it attachment services for tcp listeners.
 #  This controls services registered using bulk and auto registration.
 #
-#  remot3.it, Inc. Copyright 2019. All rights reserved.
+#  remote.it, Inc. Copyright 2018. All rights reserved.
 #
 ##### Settings #####
 VERSION=2.1.6
```
Comment (danawoodman) [inline connectd/usr/bin/connectd_startup_control]: Should this be `2.2.5`?

File: connectd/usr/bin/connectd_startup_control [hunk 3]
```diff
@@ -6,13 +6,13 @@
 #  installed remote.it attachment services for tcp listeners.
 #  This controls services registered using bulk and auto registration.
 #
-#  remot3.it, Inc. Copyright 2019. All rights reserved.
+#  remote.it, Inc. Copyright 2018. All rights reserved.
```
Comment (danawoodman) [inline connectd/usr/bin/connectd_startup_control]: This should have not changed. Legal name is `remot3.it, Inc.` and date should stay `2019`

File: connectd/usr/bin/connectd_startup_control [hunk 4]
```diff
@@ -26,22 +26,29 @@ menuLoopA()
         printf "======================= Main Menu =======================\n"
         printf "                                                         \n"
         printf "    1) Set connectd_start to run at boot (systemd)       \n"
-        printf "    2) Disable connectd_start at boot                    \n"
-        printf "    3) Exit                                              \n"
+        printf "    2) Set connectd_start to run at boot (Sys-V)         \n"
```
Comment (danawoodman) [inline connectd/usr/bin/connectd_startup_control]: Will this change require any updates to our documentation?

File: scripts/build.sh (hunk 1 above)
Comment (Digital-Larry) [reply scripts/build.sh]: Well I thought about that, however each group fully defines the variables needed prior to calling "build()".  I realize that as written, that appears redundant however the intent is to make clear that "tag" needs to be set.  My fear is that someone adding a new build target might not notice it if it was just set once and assumed to apply for all the subsequent build() calls.  There's probably a better way to refactor the whole build script to make all of these things cmd line params of the build call.

Comment (danawoodman) [reply scripts/build.sh]: I guess what I am saying is why not just make it optional/default to `""`?

**Comment by Digital-Larry** [comment]
I undid the changes adding support for the Sys-V scripts so those are gone.  

Vyos startup for bulk reg will use the Vyos-specific script as described in CTD-264.  I also got rid of the repetitive "tag" declaration but I don't know that it improved readability of the code.

File: scripts/build.sh (hunk 1 above)
Comment (Digital-Larry) [reply scripts/build.sh]: Fixed with additional changes below.

File: connectd/usr/bin/connectd_startup_control (hunk 3 above)
Comment (Digital-Larry) [reply connectd/usr/bin/connectd_startup_control]: reverted to previous version of file

File: connectd/usr/bin/connectd_startup_control (hunk 4 above)
Comment (Digital-Larry) [reply connectd/usr/bin/connectd_startup_control]: reverted to previous version of file

File: connectd/usr/bin/connectd_startup_control (hunk 2 above)
Comment (Digital-Larry) [reply connectd/usr/bin/connectd_startup_control]: Reverted to previous version of file.

**Pull request merged**